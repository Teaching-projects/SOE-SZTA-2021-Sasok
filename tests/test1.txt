150
15
1
10
15
2
30
170
13
1