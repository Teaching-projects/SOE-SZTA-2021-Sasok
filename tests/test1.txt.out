Player HP-ja: Player DMG-je: Player AS-je: Player DMG növekedése: Player HP növekedése: Player AS szorzoja: Player Szintlépéshez szükséges xp mennyisége: Második egység HP-ja: Második egység DMG-je: Második egység AS-je: 
A Csata elkezdődött! Harcos és Sámán megtámadták egymást. életük: Harcos 137, Sámán 155
Harcos megtámadta Sámán, így Sámán élete - 140
Sámán megtámadta Harcos, így Harcos élete - 124
Harcos a(z)2. szintre lépett
Harcos megtámadta Sámán, így Sámán élete - 125
Sámán megtámadta Harcos, így Harcos élete - 126
Harcos megtámadta Sámán, így Sámán élete - 100
Sámán megtámadta Harcos, így Harcos élete - 113
Harcos a(z)3. szintre lépett
Harcos megtámadta Sámán, így Sámán élete - 75
Sámán megtámadta Harcos, így Harcos élete - 115
Harcos a(z)4. szintre lépett
Harcos megtámadta Sámán, így Sámán élete - 40
Sámán megtámadta Harcos, így Harcos élete - 117
Harcos a(z)5. szintre lépett
Harcos megtámadta Sámán, így Sámán élete - -5
Sámán megtámadta Harcos, így Harcos élete - 132

Harcos GYOZEDELMESKEDETT!
